components/managedidentity/locals.tf

```diff
@@ -1,11 +1,38 @@
 locals {
-  mi_environment = var.env == "ptlsbox" ? "cftsbox-intsvc" : var.env == "ptl" ? "cftptl-intsvc" : var.env == "sbox" ? "sandbox" : var.env == "stg" ? "aat" : var.env == "dev" ? "preview" : var.env == "test" ? "perftest" : var.env
+  mi_environment         = var.env == "ptlsbox" ? "cftsbox-intsvc" : var.env == "ptl" ? "cftptl-intsvc" : var.env == "sbox" ? "sandbox" : var.env == "stg" ? "aat" : var.env == "dev" ? "preview" : var.env == "test" ? "perftest" : var.env
+  jenkins_mi_environment = var.env == "sbox" ? "sandbox" : var.env == "prod" ? "jenkins_prod" : var.env == "sandbox" ? "jenkins_sbox" : var.env
```

Comment:
```suggestion
  jenkins_mi_environment = var.env == "prod" ? "jenkins_prod" : var.env == "sbox" ? "jenkins_sbox" : var.env
```

Reply:
oh do not need to add sandbox and prod as those are just var.env

Reply:
tried to add your suggestions and got this message as cannot find it for sandbox and prod 

added mine back like this and it works

`  jenkins_mi_environment = var.env == "sbox" ? "sandbox" : var.env == "prod" ? "prod" : var.env == "prod" ? "jenkins_prod" : var.env == "sandbox" ? "jenkins_sbox" : var.env
`

Reply:
I think it is coz there is also sandbox and prod environment